x-pack/plugins/enterprise_search/public/applications/shared/not_found/not_found.tsx

```diff
@@ -37,9 +37,11 @@ interface NotFoundProps {
     ID: string;
     SUPPORT_URL: string;
   };
+  // Optional breadcrumbs
+  breadcrumbs?: string[];
```

Comment:
Something I'm becoming a big fan of in my own code for pass-through props is pulling their type directly from the child component's own props

```
import {SetPageChromeProps } from ...

interface NotFoundProps {
   breadcrumbs?: SetPageChromeProps['trail']
}
```

Not sure if it's a pattern we want to use but I've found it useful to indicate that it's a pass-through prop and guaranteeing types while I'm coding.

Reply:
I like this a lot, will add it in!

Reply:
Ah actually in this case SetPageChromeProps is already referencing a reused type called `BreadcrumbTrail` so I'll just grab that :)

Reply:
2c3423e